### PR TITLE
Adds badges to README linking to GLV repository pages

### PR DESCRIPTION
**Pull request by Cole-Greer**
Just a small update that I saw some other projects doing and I thought was cool.

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -15,7 +15,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ////
 == TinkerPop3
-image::https://codecov.io/gh/apache/tinkerpop/branch/master/graph/badge.svg?token=TojD2nR5Qd[link="https://codecov.io/gh/apache/tinkerpop"]
+image:https://img.shields.io/maven-central/v/org.apache.tinkerpop/gremlin-driver?color=brightgreen[link="https://mvnrepository.com/artifact/org.apache.tinkerpop/gremlin-driver"]
```
Comment (vkagamlyk) [inline README.asciidoc:18]: This links are to the latest 3.6.2 version, may be better to also add a link to the 3.5.5?

**Review by FlorianHockmann — APPROVED** [review]
Good idea!

VOTE +1

File: README.asciidoc (hunk 1 above)
Comment (Cole-Greer) [reply README.asciidoc:18]: From what I've seen I agree most projects tend to just link the latest version. As it's configured now, the badges will all update once a new release is published. It is possible to have it link to different versions depending on what branch you are in but I fear that may involve manually updating the badges as part of the release process (the badge providers I'm using here don't seem to support such functionality).

My preference would be to leave it with just the latest version unless there is enough demand for branch based versioning to justify the additional setup and maintenance.

Comment (vkagamlyk) [reply README.asciidoc:18]: Agreed

**Comment by xiazcy** [comment]
I like the addition. Thanks Cole! VOTE +1

**Comment by vkagamlyk** [comment]
VOTE +1

**Pull request merged**